src/main/java/ch/njol/skript/effects/EffCancelDrops.java

```diff
@@ -0,0 +1,112 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.effects;
+
+import org.bukkit.event.Event;
+import org.bukkit.event.block.BlockBreakEvent;
+import org.bukkit.event.entity.EntityDeathEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.ScriptLoader;
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Events;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+@Name("Cancel Drops")
+@Description("Cancels drops of items or experiences in a death or block break event. " +
+		"Please note that this doesn't keep items or experiences of a dead player. If you want to do that, " +
+		"use the <a href='effects.html#EffKeepInventory'>Keep Inventory / Experience</a> effect.")
+@Examples({"on death of a zombie:",
+		"\tif name of the entity is \"&cSpecial\":",
+		"\t\tcancel drops of items",
+		"",
+		"on break of a coal ore:",
+		"\tcancel the experience drops"})
+@Since("INSERT VERSION")
+@RequiredPlugins("1.12.2 or newer (cancelling item drops of blocks)")
+@Events({"death", "break / mine"})
+public class EffCancelDrops extends Effect {
+
+	private static final boolean CAN_CANCEL_BLOCK_ITEM_DROPS = Skript.methodExists(BlockBreakEvent.class, "setDropItems", boolean.class);
```

Comment:
```suggestion
	private static final boolean CANCELLABLE_DROPS = Skript.methodExists(BlockBreakEvent.class, "setDropItems", boolean.class);
```

Reply:
Not descriptive because it is for a method to cancel *block* - *item* drops, maybe `CANCELLABLE_BLOCK_ITEM_DROPS` but why